.github/workflows/integrate.yaml

```diff
@@ -96,7 +101,7 @@ jobs:
 
     - name: Run test
       run: |
-        sg microk8s -c "microk8s enable ingress metallb:10.64.140.43-10.64.140.49"
+        sg microk8s -c "microk8s enable storage rbac ingress metallb:10.64.140.43-10.64.140.49"
```

Comment:
I think this can be removed and replaced by using the `microk8s-addons` argument like done in the charm integration tests